pkg/apis/ingress/v1alpha1/clusteringress_types.go

```diff
@@ -75,8 +78,15 @@ type ClusterIngressHighAvailability struct {
 
 // ClusterIngressStatus defines the observed state of ClusterIngress
 type ClusterIngressStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+
+	// Replicas is the actual number of observed router instances.
+	Replicas int32 `json:"replicas"`
+
+	// Selector is a label selector, in string format, for pods
+	// corresponding to the ClusterIngress. The number of matching pods
+	// should equal the value of replicas.
+	Selector string `json:"labelSelector"`
```

Comment:
Can you help me understand the rationale for this field?

Reply:
It is required by the HPA.  https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#scale-subresource

Reply:
Excellent, thanks